File: 2.0/java/contacts/findContacts.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.contacts.model.Contact;
import com.callfire.api.client.api.contacts.model.request.FindContactsRequest;

import java.util.Arrays;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        FindContactsRequest request = FindContactsRequest.create()
            .id(Arrays.asList(11646003L, 12646003L, 13776003L))
            .number(Arrays.asList("12135551126", "12136666123"))
            .contactListId(14400809003L)
            .propertyName("14400809003")
            .propertyValue("14400809003")
            .offset(0L)
            .limit(10L)
            .build();
        Page<Contact> contacts = client.contactsApi().find(request);
    }
}

File: 2.0/csharp/contacts/findContacts.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Contacts.Model;
using CallfireApiClient.Api.Contacts.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request = new FindContactsRequest
        {
            Id = new List<long> { 11646003, 12646003, 13776003 },
            Number = new List<string> { "12135551126", "12136666123" },
            ContactListId = 14400809003,
            PropertyName = "14400809003",
            PropertyValue = "14400809003",
            Offset = 0,
            Limit = 10,
            Fields = "items(id,name,size)"
        };
        Page<Contact> contacts = client.ContactsApi.Find(request);
    }
}
